apps/meteor/client/views/oauth/hooks/useOAuthAppQuery.ts

```diff
@@ -19,11 +19,7 @@ export const useOAuthAppQuery = (clientId: string | undefined, options?: UseOAut
 			}
 
 			const { oauthApp } = await getOAuthApp({ clientId });
-			return {
-				...oauthApp,
-				_createdAt: new Date(oauthApp._createdAt),
-				_updatedAt: new Date(oauthApp._updatedAt),
-			};
+			return oauthApp;
```

Comment:
Are those dates being used somewhere? If so maybe endpoint should return them.

Reply:
No, this data is not used anywhere else. The `useOAuthAppQuery` function is [called](https://github.com/RocketChat/Rocket.Chat/blob/1b072df153744b0f9e80044a2279abb88bed000d/apps/meteor/client/views/oauth/OAuthAuthorizationPage.tsx#L16) only during the rendering of the consent screen. The result of the call is [passed](https://github.com/RocketChat/Rocket.Chat/blob/1b072df153744b0f9e80044a2279abb88bed000d/apps/meteor/client/views/oauth/OAuthAuthorizationPage.tsx#L32C10-L32C31) to `AuthorizationFormPage`.

From the passed object, only:
 - [clientId](https://github.com/RocketChat/Rocket.Chat/blob/1b072df153744b0f9e80044a2279abb88bed000d/apps/meteor/client/views/oauth/components/AuthorizationFormPage.tsx#L45)
 - [clientId](https://github.com/RocketChat/Rocket.Chat/blob/1b072df153744b0f9e80044a2279abb88bed000d/apps/meteor/client/views/oauth/components/AuthorizationFormPage.tsx#L51)
 - [clientId](https://github.com/RocketChat/Rocket.Chat/blob/1b072df153744b0f9e80044a2279abb88bed000d/apps/meteor/client/views/oauth/components/AuthorizationFormPage.tsx#L74C51-L74C59)
 - [name](https://github.com/RocketChat/Rocket.Chat/blob/1b072df153744b0f9e80044a2279abb88bed000d/apps/meteor/client/views/oauth/components/AuthorizationFormPage.tsx#L65) are used.